keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,389 @@
+
+In theory a reduced release cadence will cause more changes for every release.
+This means that there will be an increased risk, which would usually be split
+up into 4 dedicated milestones rather than 3.
+
+SIG Release cannot mitigate this risk directly, but is able to track and
+influence it during each release cycle. It's the responsibility of SIG Release,
+together with SIG Testing and SIG Architecture, to identify new gaps and issues
+in the release cadence and mitigate them on a case-by-case basis.
+
+#### Attention to tests
+
+This KEP does not propose any change to the release cycle itself and assumes
+that the same periods for Code and Test Freeze. Assuming that, there is an
+increased risk for flakes and test failures. It will be the responsibility of
+SIG Release to mitigate this, together with the CI signal role. If we speak
+about an overall release cycle enhancement of 3-4 weeks, then we believe that
+SIG Release is able to mitigate this risk over multiple releases.
+
+#### Attention to dependencies
+
+Having fewer releases will introduce the risk of missing dependencies — for
+example, Golang upgrades. This has to be mitigated on a case-by-case basis, in
+the same way as it is being done right now.
+
+#### Feature graduation
+
+Research discovered that only 5% of Kubernetes features advanced from Alpha to
+GA in the minimum 3 releases. However, the same research showed that reminders
+from the Release Team played a critical role in advancement of more than 50% of
+features. With a longer release cycle, this reminder activity can be
+expected to slow down. As such, advancement will need to be mitigated by making
+sure that SIGs keep track of their feature enhancement in more detail.
+
+## Design Details
+
+### Schedule Policy
+
+The exact details of the release schedule are created by the team leads before
+the actual cycle begins. SIG Release only provides a lightweight policy, which
+is defined as:
+
+1. The first Kubernetes release of a year should start at the second or third
+   week of January to provide people more room after coming back from the
+   Christmas holidays.
+
+2. The last Kubernetes release of a year should be finished by the middle of
+   December.
+
+3. A Kubernetes release cycle has a length of of ~15 weeks.
+
+4. Events like KubeCon will be considered as blocked from development or
+   decision-making from the SIG release perspective. SIG Release will also
+   consider the week before and after the event in the same way.
+
+5. An explicit SIG release break of at least two weeks between each cycle will
+   be enforced.
+
+   This does not mean that zero development can happen during that time.
+   Rather, SIG Release will use this time to do the release retrospective and
+   plan for the next cycle.
+
+### Feedback survey
+
+SIG Release will draft an experience survey after the first three releases from
+which the new cadence has been applied. This survey which will include questions
```

Comment:
As noted above I think the timing of the feedback survey needs to be fleshed out a bit.  I worry about it happening between releases (does that mean it's going to affect the next one? or the one after for a notice period?). 

Also if the new cadence begins with 1.22, that would mean we'd potentially be changing cadence back in mid 2022? Feels like in that case taking stock towards the end of 1.23 (EOY release) would make more sense to move forward in 2023 with a solid plan for the entire year?